lib/utils.js

```diff
@@ -13,6 +13,7 @@ const du = require('du');          // Get amount used
 const disk = require('diskusage'); // Get amount free
 const assert = require('assert');
 const bytes = require('bytes');
+const web3utils = require('web3-utils');
```

Comment:
Where is the github for this? The npm page goes to a 404

Reply:
Here: https://github.com/ethereum/web3.js/tree/1.0/packages/web3-utils

Reply:
K, thanks. Looks like this is the code for isAddress https://github.com/ethereum/web3.js/blob/1.0/packages/web3-utils/src/utils.js#L87

Reply:
The important part is checkAddressChecksum that gets called at the end of isAddress
https://github.com/ethereum/web3.js/blob/1.0/packages/web3-utils/src/utils.js#L109